### PR TITLE
Fixed buggs and some adjustments

### PR DESCRIPTION
**Pull request by UrosLITS**
Minor bug fixes and some adjustments inside the code.

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -61,7 +61,7 @@
   "missing_special_char": "Password must contain at least one special character",
   "not_long_enough_pass": "Password must contain at least 8 characters",
   "error": "Can not  reach the server",
-  "error_auth": "Incorrect email",
+  "error_auth": "Incorrect email or  password",
```
Comment (petarLits) [inline lib/l10n/app_en.arb:64]: what is with translation?

**Review by petarLits — CHANGES_REQUESTED** [review]

**Review by milicaLits — APPROVED** [review]

File: lib/l10n/app_sr.arb [hunk 2]
```diff
@@ -60,7 +60,7 @@
   "missing_special_char": "Sifra mora da sadrzi makar jedan specijalni karakter",
   "not_long_enough_pass": "Sifra mora da sadrzi makar  karaktera",
   "error": "Izgubljena konekcija sa serverom",
-  "error_auth": "Neispravan email",
+  "error_auth": "Neispravan email ili password",
```
Comment (petarLits) [inline lib/l10n/app_sr.arb]: passsword is not serbian word

**Review by petarLits — CHANGES_REQUESTED** [review]

**Review by milicaLits — APPROVED** [review]

File: lib/data/firebase_cloud_messaging.dart [hunk 3]
```diff
@@ -113,4 +111,10 @@ class FCM {
       );
     }
   }
+
+  Future<void> _initAccountCredentials() async {
+    final jsonCredentials =
+        await rootBundle.loadString('assets/book-d21ec-c64560991d06.json');
```
Comment (milicaLits) [inline lib/data/firebase_cloud_messaging.dart]: use constant

File: lib/presentation/bookPages/bloc/book_bloc.dart [hunk 4]
```diff
@@ -41,7 +43,9 @@ class BookBloc extends Bloc<BookEvents, BookState> {
 
     if (currentPageIndex < book.bookData!.pages.length - 1) currentPageIndex++;
     emit(DisplayBookPageState(
-        bookData: book.bookData!, pageIndex: currentPageIndex));
+        bookData: book.bookData!,
+        pageIndex: currentPageIndex,
+        doNotPopDialog: true));
```
Comment (milicaLits) [inline lib/presentation/bookPages/bloc/book_bloc.dart]: remove this

File: lib/presentation/bookPages/bloc/book_bloc.dart [hunk 5]
```diff
@@ -69,27 +71,29 @@ class BookBloc extends Bloc<BookEvents, BookState> {
       currentPageIndex--;
     }
     emit(DisplayBookPageState(
-        bookData: book.bookData!, pageIndex: currentPageIndex));
+        bookData: book.bookData!,
+        pageIndex: currentPageIndex,
+        doNotPopDialog: true));
```
Comment (milicaLits) [inline lib/presentation/bookPages/bloc/book_bloc.dart]: ditto

File: lib/data/firebase_auth/firebase_auth_singleton.dart [hunk 6]
```diff
@@ -29,7 +29,7 @@ class FirebaseAuthSingleton {
   Future<void> login(String email, String password) async {
     await _auth
         .signInWithEmailAndPassword(email: email, password: password)
-        .timeout(Duration(seconds: 3), onTimeout: () {
+        .timeout(Duration(seconds: timeoutDuration), onTimeout: () {
       throw Exception(timeoutErrorMessage);
```
Comment (milicaLits) [inline lib/data/firebase_auth/firebase_auth_singleton.dart]: throw custom Exception class

**Review by milicaLits — CHANGES_REQUESTED** [review]

File: lib/data/firebase_cloud_messaging.dart [hunk 7]
```diff
@@ -78,7 +76,7 @@ class FCM {
 
     client.close();
     if (response.statusCode == 200) {
-      return true; // Success!
+      return true;
     }
 
     print('Notification Sending Error Response status: ${response.statusCode}');
```
Comment (petarLits) [inline lib/data/firebase_cloud_messaging.dart]: do we need this?

**Review by petarLits — CHANGES_REQUESTED** [review]

**Review by petarLits — APPROVED** [review]

**Review by milicaLits — APPROVED** [review]

**Pull request merged**